libraries/protocol/include/steem/protocol/config.hpp

```diff
@@ -44,7 +44,7 @@
 
 #else // IS LIVE STEEM NETWORK
 
-#define STEEM_BLOCKCHAIN_VERSION              ( version(0, 22, 1) )
+#define STEEM_BLOCKCHAIN_VERSION              ( version(0, 22, 5) )
```

Comment:
Why are you trying to change the blockchain version? It doesn't need to be bumped for a change like this.